docs/guides/admin/docs/modules/searchindex/index.md

```diff
@@ -21,4 +21,4 @@ Opencast currently uses two search index types: Solr and Elasticsearch.
 
 - Elasticsearch is the back-end for both the administrative user interface as well as the external API.
 
-    [Elasticsearch Configuration Guide](elasticsearch.md)
+    [Elasticsearch Configuration Guide](../../configuration/elasticsearch.md)
```

Comment:
Same here: If it's actually configuration, it shouldn't be listed here.

Reply:
I'll remove it, you're right.